mkosi/__init__.py

```diff
@@ -1814,6 +1834,14 @@ def install_type1(
 
     kmods = build_kernel_modules_initrd(context, kver)
 
+    dtb = None
+    if context.config.devicetree:
+        dtb_src = find_requested_devicetree(context, kver)
+
+        dtb = dst / context.config.devicetree
+        dtb.parent.mkdir(parents=True, exist_ok=True)
```

Comment:
This can be dropped, we already create the parent directory earlier in this function

Reply:
We create "dst" previously, but context.config.devicetree typically contains a vendor path in it (such as qcom/sc8280xp-crd.dtb, someone need to create the "qcom" part)

But looking at the code again, I think the mkdir and the copy should be done with umask...

Reply:
Ahah, in that case, like you already said, please do it with umask